pkg/core/region_tree.go

```diff
@@ -342,3 +346,20 @@ func (t *regionTree) TotalWriteRate() (bytesRate, keysRate float64) {
 	}
 	return t.totalWriteBytesRate, t.totalWriteKeysRate
 }
+
+func (t *regionTree) AtomicAddStaleRegionCnt() {
+	if t.length() == 0 {
```

Comment:
Why do we need to check the length here?

Reply:
Now we do not need to check it after moving `staleRegionCnt` into `regionsInfo`